pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -383,9 +383,13 @@ def refresh(self, mountpoint, device, rootName, snapshots=False):
 
         if rootName and "_" in rootName:
             rootName = rootName.replace("_", "__")
+        delete_all_text = (C_("GUI|Custom Partitioning|Confirm Delete Dialog",
+                               "Delete _all other file systems in the %s root as well.") +
+                           "\n" +
+                           C_("GUI|Custom Partitioning|Confirm Delete Dialog",
+                               "(Including those used by other installed operating systems.)"))
```

Comment:
It would be a bit more natural sounding if the second string began "This includes those used by..."